### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#191] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-10-28 110455](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/92cd8bd5-d68a-4b4d-8f84-cc02b0986515)

**Comment by IonutCicio** [comment]
_(Nota: ho usato la notazione_ $\equiv _n$ _per indicare la congruenza modulo_ $n$ _e la notazione_ $(a, b)$ per indicare $MCD(a, b)$ _)._ 

Prima di tutto, bisogna determinare se il sistema è risolvibile, quindi determinare se, per ogni congruenza nella forma $ax \equiv _n b$, $(a, n)$ divide $b$

$$(4, 22) = 2 \mid 2$$

$$(3, 7) = 1 \mid 2$$

Il sistema è risolvibile, e si può ridurre a un sistema cinese dato che $(22, 7) = 1$ ($22$ e $7$ sono coprimi). Prima di tutto si riducono le equazioni

$$
\begin{cases}
2x \equiv _{11} 1\\
3x \equiv _7 2\\
\end{cases}
$$

Ora è necessario trovare l'inverso di $2 \\ mod \\ 11$ e l'inverso di $3 \\ mod \\ 7$

$$2 \bar{x} \equiv _{11} 1 \implies \bar{x} \equiv _{11} 6$$

$$3 \bar{x} \equiv _7 1 \implies \bar{x} \equiv _7 5$$

Moltiplicando ambo i membri di ciascuna equazione per il rispettivo inverso, si ottiene un sistema cinese

$$
\begin{cases}
6 \cdot 2x \equiv _{11} 6 \cdot 1\\
5 \cdot 3x \equiv _7 5 \cdot 2\\
\end{cases}
\implies
\begin{cases}
x \equiv _{11} 6\\
x \equiv _7 10\\
\end{cases}
\implies
\begin{cases}
x \equiv _{11} 6\\
x \equiv _7 3\\
\end{cases}
$$

È noto che la soluzione del sistema esiste, è unica ed è nella forma

$$
\tilde{x} = R_1\tilde{x}_1 + R_2\tilde{x}_2
$$

Dove, dato $R = 11 \cdot 7 = 77$, si calcolano

$$R_1 = \frac{R}{11} = 7$$

$$R_2 = \frac{R}{7} = 11$$

Ora si possono trovare $\tilde{x} _1, \tilde{x} _2$

$$R_1 \tilde{x}_1 \equiv _{11} 6 \implies 7 \tilde{x}_1 \equiv _{11} 6 \implies \tilde{x}_1 \equiv _{11} 4$$

$$R_2 \tilde{x}_2 \equiv _7 3 \implies 11 \tilde{x}_2 \equiv _7 3 \implies 4 \tilde{x}_2 \equiv _7 3 \implies \tilde{x}_2 \equiv _7 6$$

Quindi $\tilde{x} = 7 \cdot 4 + 11 \cdot 6 \equiv _{77} 94 \equiv _{77} 17$.

**Comment by CiottoloMaggico** [comment]
<h1>Utilizzando dimostrazione metodo cinese</h1>
<br>
<img src="https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/d52ad57c-bc74-4a58-aad2-99c9e6c7957b"/>
<br>
<h1>Utilizzando sostituzione</h1>
<img src="https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/7b395c6c-c57e-4986-96af-ee00a6f659f8"/>
